x-pack/test/alerting_api_integration/security_and_spaces/tests/actions/builtin_action_types/es_index_preconfigured.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+// from: x-pack/test/alerting_api_integration/common/config.ts
+const ACTION_ID = 'preconfigured-es-index-action';
+const ES_TEST_INDEX_NAME = 'functional-test-actions-index-preconfigured';
+
+// eslint-disable-next-line import/no-default-export
+export default function indexTest({ getService }: FtrProviderContext) {
```

Comment:
This file was copied from `es_index.ts`, but only tests a pre-configured action, and only that you can execute it.